### PR TITLE
Launch Game Lab into public beta

### PR DESCRIPTION
**Pull request by caleybrock**
Create code.org/educate/gamelab to get ready for launch: This isn't perfect, but I want to get it in staging so Poorva could update links if she needs to. Most of this was copied from code.org/educate/applab

- Create page for code.org/educate/gamelab
- Have code.org/gamelab redirect to code.org/educate/gamelab
- Have "Make my own" button in Game Lab go to code.org/educate/gamelab
- Rename "Challenges" to "Sample Projects" and add button that links to project gallery 

screencapture-localhost-code-org-3000-educate-gamelab-1495841091616

cc @poorvasingal

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/gamelab.haml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  - csdunit3_videos << { id: 'csd_gamelab_drawloop', category_title: 'Draw loop', code: 'G6QJeuHhqCM', download_path: 'https://videos.code.org/cs-discoveries/gamelab/gamelab_drawloop.mp4', studio_stage: '/s/csd3/stage/7/puzzle/2' }
+  - csdunit3_videos << { id: 'csp_applab_conditionals_1', category_title: 'Conditionals', code: 'y3rCKJNOwpA', download_path: 'https://videos.code.org/2015/csp/applab/conditionals_1.mp4', studio_stage: '/s/csd3/stage/11/puzzle/3' }
+  - csdunit3_videos << { id: 'csp_applab_conditionals_2', code: 'UDi7xgIIW8E', download_path: 'https://videos.code.org/2015/csp/applab/conditionals_2.mp4', studio_stage: '/s/csd3/stage/13/puzzle/4' }
+  - csdunit3_videos << { id: 'csd_gamelab_velocity', category_title: 'Sprite velocity', code: 'zcP94yMYqrI', download_path: 'https://videos.code.org/cs-discoveries/gamelab/gamelab_velocity.mp4', studio_stage: '/s/csd3/stage/15/puzzle/2' }
+
+  - csdunit3_videos.each do |video|
+    // Show the chapter title at the
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/gamelab.haml]: nit: need to line-wrap this early?

File: pegasus/sites.v3/code.org/public/educate/gamelab.haml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      .clear
+      %h2= video[:category_title]
+    %div{style: "float:left; padding:10px; width: 275px;"}
+      = view :display_video_thumbnail, id: video[:id],  video_code:video[:code], caption: video[:caption], play_button: 'center', download_path: video[:download_path], letterbox: 'false'
+      - unless video[:studio_stage].nil_or_empty?
+        %a{href: CDO.studio_url(video[:studio_stage])} Try it now
+
+:javascript
+  $(function() {
+    $('.highlight-item').mouseover(function() {
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/gamelab.haml]: maybe a comment about what exactly is fading in & out?  also do we need a touch equivalent for non-mouse users?

**Comment by caleybrock** [comment]
fyi @poorvasingal @tanyaparker - this should be ready to go.

File: pegasus/sites.v3/code.org/public/educate/gamelab.haml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  %h2 Explore these sample projects and take the challenge to make them even better
+  .highlights-content
+    - cta = "Remix the project"
+    = view :highlight_item, item: {image: "game-lab/example-sunset", title: "Sunset shapes", desc: "Watch the sun set. Challenge: Make the sky go dark once the sun has set.", cta: cta, url: 'https://studio.code.org/projects/gamelab/Z9QRSmJBfNY25T3ViLp0MTy4uAuXNNXpZrmhPsUdEro/remix'}
+    = view :highlight_item, item: {image: "game-lab/example-trees", title: "Make your own character", desc: "Use your arrow keys to move the character around. Challenge: add more to the scene and make the character look like you.", cta: cta, url: 'https://studio.code.org/projects/gamelab/XVEFITms8-c7_F-ND-FKL8w3A-oEpFD_GKAJCB2VPCQ/remix'}
+    = view :highlight_item, item: {image: "game-lab/example-bounce", title: "Bounce", desc: "Move the platform to avoid the ball hitting the bottom. Challenge: Add different obstacles and points", cta: cta, url: 'https://studio.code.org/projects/gamelab/J2R-PHcvE4Wgb2xO2bjk0Q/remix'}
+  .clear
+  %a{:href=>CDO.studio_url("/projects/public")}
+    %button.default See more student projects in the public gallery
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/educate/gamelab.haml]: Would prefer this shorter like "View project gallery"

File: pegasus/sites.v3/code.org/public/educate/gamelab.haml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+title: Game Lab
+theme: responsive
+video_player: true
+---
+
+%link{href: "/css/jumbotron-banner.css", rel: "stylesheet"}
+%link{href: "/shared/css/course-blocks.css", rel: "stylesheet"}
+%link{href: "/css/highlights.css", rel: "stylesheet"}
+
+= view :jumbotron_banner, img: CDO.code_org_url("/images/app-lab/girl-cropped.jpg"), title: "Game Lab (beta)", desc: "Game Lab is a programming environment where you can make simple animations and games with objects and characters that interact with each other. Design an animation or game, code with blocks or JavaScript to make it work, then share your app in seconds.", specs: "Ages 13+, all modern browsers, English only", url: CDO.studio_url('/p/gamelab'), cta: "Make an animation or game", photo_cred: "Photo courtesy of Fundación Sadosky", nav_bar: [{text: 'Resources', url: '#resources'}, {text: 'Challenges', url: '#challenges'}, {text: 'Video Library', url: '#videos'}]
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/educate/gamelab.haml]: Rename `Challenges` here to `Sample projects`

File: pegasus/sites.v3/code.org/public/educate/gamelab.haml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    = view :highlight_item, item: {image: "game-lab/example-trees", title: "Make your own character", desc: "Use your arrow keys to move the character around. Challenge: add more to the scene and make the character look like you.", cta: cta, url: 'https://studio.code.org/projects/gamelab/XVEFITms8-c7_F-ND-FKL8w3A-oEpFD_GKAJCB2VPCQ/remix'}
+    = view :highlight_item, item: {image: "game-lab/example-bounce", title: "Bounce", desc: "Move the platform to avoid the ball hitting the bottom. Challenge: Add different obstacles and points", cta: cta, url: 'https://studio.code.org/projects/gamelab/J2R-PHcvE4Wgb2xO2bjk0Q/remix'}
+  .clear
+  %a{:href=>CDO.studio_url("/projects/public")}
+    %button.default See more student projects in the public gallery
+
+#videos
+  %h1.responsive-body Video library
+  Some of these videos appear in Unit 3 (Animations and Games) of the CS Discoveries course and are displayed here as a quick introduction to each of the main concepts from the unit. Check out each video to see how you can add new features to your project.
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/educate/gamelab.haml]: Can you hyperlink `CS Discoveries course` to /educate/csd?

**Pull request merged**